UI/TouchControlLayoutScreen.cpp

```diff
@@ -612,7 +619,8 @@ void TouchControlLayoutScreen::CreateViews() {
 	tabHolder->SetTag("TouchControlLayout");
 	root_->Add(tabHolder);
 
-	layoutView_ = root_->Add(new ControlLayoutView(new AnchorLayoutParams(leftColumnWidth + 10, 0.0f, 0.0f, 0.0f, false)));
+	root_->Add(new ItemHeader("", new AnchorLayoutParams(leftColumnWidth + 10, bounds.h*(1.0-layoutAreaScale)-40, NONE, NONE, false)));
```

Comment:
Hm, it's a bit weird to use an ItemHeader, might be better to use a spacer or something.

-[Unknown]

Reply:
Mhh, I was wondering to use it to add a tip like: "You may edit visibility and add additional controls pressing the Customize button on the left" as that screen might get missed.

Tho' I was not really sure how to write it down and then I forgot :P